src/modules/shopPage/components/FilterMenu/FilterShortMenu/FilterShortMenuContainer.tsx

```diff
@@ -0,0 +1,156 @@
+import { useDispatch, useSelector } from "react-redux";
+import { actions } from "../../../reducer";
+import {
+  selectSelectedBrand,
+  selectSelectedDiametr,
+  selectSelectedPrice,
+  selectSelectedProfile,
+  selectSelectedSeason,
+  selectSelectedWidth,
+} from "../../../selectors";
+import { selectFilterData } from "../../../../mainPage/selectors";
+
+import FilterShortMenuRow from "./FilterShortMenuRow";
+import FilterShortMenuColumnPrice from "./FilterShortMenuColumnPrice";
+import FilterShortMenuColumn from "./FilterShortMenuColumn";
+import FilterShortMenuReset from "./FilterShortMenuReset";
+
+import { ButtonGroup } from "@mui/material";
+
+import WidthIcon from "../../../../../shared/components/Icons/WidthIcon";
+import ProfileIcon from "../../../../../shared/components/Icons/ProfileIcon";
+import DiametrIcon from "../../../../../shared/components/Icons/DiametrIcon";
+import PriceIcon from "../../../../../shared/components/Icons/PriceIcon";
+import SeasonIcon from "../../../../../shared/components/Icons/SeasonIcon";
+import BrandIcon from "../../../../../shared/components/Icons/BrandIcon";
+import ResetIcon from "../../../../../shared/components/Icons/ResetIcon";
+import { PayloadAction } from "typesafe-actions";
+import { PayloadAction as PayloadActionRedux } from "@reduxjs/toolkit";
+
+const FilterShortMenuContainer = () => {
+  const dispatch = useDispatch();
+  const selectWidth = useSelector(selectSelectedWidth);
+  const selectProfile = useSelector(selectSelectedProfile);
+  const selectDiametr = useSelector(selectSelectedDiametr);
+  const selectedPrice = useSelector(selectSelectedPrice);
+  const selectedSeason = useSelector(selectSelectedSeason);
+  const selectedBrand = useSelector(selectSelectedBrand);
+  const filtersParams = useSelector(selectFilterData());
+  const minPrice = Math.min(...filtersParams.prices);
+  const maxPrice = Math.max(...filtersParams.prices);
+
+  const visableResetButton =
+    selectedPrice[0] !== minPrice ||
+    selectedPrice[1] !== maxPrice ||
+    selectWidth.length > 0 ||
+    selectProfile.length > 0 ||
+    selectDiametr.length > 0 ||
+    selectedSeason.length > 0 ||
+    selectedBrand.length > 0;
+
+  function handleClearRowsFilters() {
+    return (
+      e: React.MouseEvent<HTMLDivElement, MouseEvent>,
+      filterAction: () => PayloadAction<string, void>
+    ) => {
+      e.stopPropagation();
+      dispatch(filterAction());
+    };
+  }
```

Comment:
did you try to do a closure here?) use an arrow function instead

Reply:
Done